README.md

```diff
@@ -2,53 +2,199 @@
 
 # NetKit.Chat
 
-Open source product is used to communicate people in real time. 
-This project uses ASP.NET Core 2.0
+NetKit.Chat is open source product which used to communicate people in real time. Chat provide you ability to send group and direct messages including images, manage groups and receive push notifications.
```

Comment:
NetKit.Chat is open source product which used to add communication feature into your solution. Chat provides you ability to send group and direct messages including images, manage groups and receive push notifications.